pelita/maze_generator.py

```diff
@@ -55,21 +63,17 @@ def sample_nodes(nodes, k, rng=None):
         return nodes
 
 
-def find_trapped_tiles(graph, width, include_chambers=False):
+def find_trapped_tiles(graph, gaps, include_chambers=False):
```

Comment:
I think that `gaps` is not a good name for the argument, or we need a doc-string that explains what it means. In general, maybe this could be by default `None`, so that `find_chambers` can be tested and used independently of our particular use case? Not sure if the generalization is useful, but still thinking about us in 5 years looking back, I think it will be odd if we don't explain this or make it general.